plugins/woocommerce/src/Blocks/BlockTypes/AddToCartForm.php

```diff
@@ -128,6 +131,7 @@ protected function render( $attributes, $content, $block ) {
 
 		$previous_product = $product;
 		$product          = wc_get_product( $post_id );
+		$product_name     = $product->get_name();
```

Comment:
Should this go after we check that `$product` is an instance of `WC_Product`? Otherwise it might cause an error, no?

Even better, I wonder if we could just move it right before where we need it (L160):

```PHP
$product_name = $product->get_name();
$product      = $is_stepper_style ? $this->add_steppers( $product, $product_name ) : $product;
```